### PR TITLE
add unicode language support to save_json()

### PR DESCRIPTION
**Pull request by iwaitu**
save_json() can support unicode language.

### CODE REVIEW

**Comment by fcakyon** [comment]
Thanks a lot for your contributions!

**Comment by fcakyon** [comment]
@iwaitu can you please reformat your code by following the steps given here: https://github.com/obss/sahi#contributing

File: sahi/utils/file.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ def save_json(data, save_path):
 
     # export as json
     with open(save_path, "w", encoding="utf-8") as outfile:
-        json.dump(data, outfile, separators=(",", ":"), cls=NumpyEncoder)
+        json.dump(data, outfile, separators=(",", ":"), cls=NumpyEncoder,ensure_ascii=False)
```
Comment (devrimcavusoglu) [inline sahi/utils/file.py:42]: This may not be always wanted, I think it'd be more proper if we add this, ass an argument to `save_json()` function.

Comment (devrimcavusoglu) [reply sahi/utils/file.py:42]: @fcakyon I'm not sure if in sahi we could potentially use `ensure_ascii=False` anywhere, but making it a function argument and let user decide is more generic and better option IMO (to be on safe-side). There's a [discussion in SO](https://stackoverflow.com/q/40412714) about the usage.

Comment (fcakyon) [reply sahi/utils/file.py:42]: yes it seems in most cases `ensure_ascii=False` cause trouble.

**Pull request closed without merging**